workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
+
+    def get_form(self, *args, **kwargs):
+        """Disable fields that are pre-populated."""
+        form = super().get_form(*args, **kwargs)
+        for field in self.get_initial():
+            try:
+                form.fields[field].disabled = True
```

Comment:
>It seems that this fix wasn't released yet (django-selectable 0.9 is from 2014), so we can't fix it by upgrading django-selectable.

We can instruct pip to fetch the `develop` branch of the repository so that we have the fix. Is that fine?